src/android/toga_android/window.py

```diff
@@ -27,12 +31,15 @@ def set_app(self, app):
     def set_content(self, widget):
         # Set the widget's viewport to be based on the window's content.
         widget.viewport = AndroidViewport(widget.native)
-        # Set the app's entire contentView to this window. This means that calling
-        # Window.set_content() on any Window object automatically updates the app,
-        # meaning that every Window object acts as the MainWindow.
-        self.app.native.setContentView(widget.native)
-
-        # Attach child widgets to the this window as their container.
+        # Set the app's entire contentView to a scrollable view over the desired
+        # widget. This means that calling Window.set_content() on any Window
+        # object automatically updates the app, meaning that every Window
+        # object acts as the MainWindow.
+        scroll_view = ScrollView(MainActivity.singletonThis)
```

Comment:
I missed this in my previous review, but it's a problem for layout, as it means *every* app will be scrolling. Is this necessary for rendering purposes?

Reply:
If the underlying view doesn't need to scroll, it doesn't actually scroll. So IMHO it's not a problem.

I'm open to reconsidering using `ScrollView` here, but I don't have any Toga code that exercises any (hypothetical?) bad behavior, so I like this as-is, personally. (One day I may, and at that time, I'd be sure to fix it.)

Reply:
Done! See latest commit!